### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by alex** [Lukasa/language-restructuredtext#15]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app/src/package.js:565:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app/src/package.js:187:43)
```

**Comment by Lukasa** [comment]
=P Looks like I'll need to work out how the hell Atom wants me to do this stuff now.

**Comment by Lukasa** [comment]
Should be resolved in v0.5.0.
